frontend/src/components/profile-happening-preview.tsx

```diff
@@ -0,0 +1,103 @@
+import { Box, Flex, LinkBox, LinkOverlay, Spacer, Stack, Text, useColorModeValue } from '@chakra-ui/react';
+import NextLink from 'next/link';
+import Image from 'next/image';
+import { format, isToday } from 'date-fns';
+import { nb, enUS } from 'date-fns/locale';
+import { BiCalendar } from 'react-icons/bi';
+import type { Happening } from '@api/happening';
+import useLanguage from '@hooks/use-language';
+
+interface Props {
+    isBedpres: boolean;
+    event: Happening;
+}
```

Comment:
Kan bruker `event.happeningType` for å se om det er bedpres eller ikke.

```suggestion
interface Props {
    event: Happening;
}
```